checkpoints/eight.patch

```diff
@@ -0,0 +1,67 @@
+diff --git a/frontend/src/components/TweetFeed.jsx b/frontend/src/components/TweetFeed.jsx
+index c1b8d6d..93fc78f 100644
+--- a/frontend/src/components/TweetFeed.jsx
++++ b/frontend/src/components/TweetFeed.jsx
+@@ -2,7 +2,7 @@ import Tweet from "@/components/Tweet/SingleTweet";
+ import { Bars } from "react-loader-spinner";
+ 
+ export default function TweetFeed({ tweets, loading }) {
+-  if (!tweets.length) {
++  if (/* Task 4: Write a JavaScript expression here to check if tweets are empty */) {
+     return <div className="text-center my-8">No tweets to show right now!</div>;
+   }
+ 
+@@ -24,11 +24,12 @@ export default function TweetFeed({ tweets, loading }) {
+ 
+   return (
+     <>
+-      {tweets.map((tweet, idx) => {
++      {tweets.map((tweet) => {
+         return (
+-          <div key={tweet.id}>
+-            <Tweet tweet={tweet} /> <hr />
+-          </div>
++          // Task 5:
++          // We want to "map" each tweet to the Tweet UI component here.
```

Comment:
We want to have UI corresponding to each tweet - an array of components corresponding with the array of tweets. Which JS Array function does this?